app/lib/utils/string.js

```diff
@@ -0,0 +1,3 @@
+export function escapeShellArgument (text) {
```

Comment:
I haven't find a module that only checks the octal permissions without doing the fs.stat of a file, but maybe I missed it :)